### PR TITLE
ode 0.14

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

**Comment by nijikon** [comment]
I incorporated the drawstuff changes, so we can work on this to fix it.

**Comment by nijikon** [comment]
Homebrew/homebrew#44265

**Comment by nijikon** [comment]
Homebrew/homebrew#43312

File: Library/Formula/ode.rb [hunk 1]
```diff
@@ -12,36 +13,30 @@ class Ode < Formula
...
-
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-  end
-
   option "with-double-precision", "Compile ODE with double precision"
   option "with-shared", "Compile ODE with shared library support"
   option "with-libccd", "Enable all libccd colliders (except box-cylinder)"
+  option "with-drawstuff", "Include the drawstuff library and demos. Requires X11."
```
Comment (DomT4) [inline Library/Formula/ode.rb]: Would prefer this to be `"with-x11", "Build the drawstuff library and demos"`

File: Library/Formula/ode.rb [hunk 2]
```diff
@@ -12,36 +13,30 @@ class Ode < Formula
...
 
   deprecated_option "enable-double-precision" => "with-double-precision"
   deprecated_option "enable-shared" => "with-shared"
   deprecated_option "enable-libccd" => "with-libccd"
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
   depends_on "pkg-config" => :build
+  depends_on :x11 if build.with? "drawstuff"
```
Comment (DomT4) [inline Library/Formula/ode.rb]: And then this to just be `=> :optional`

**Comment by nijikon** [comment]
@DomT4 done

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/2a8b61f1217651bacdba8b9047d8a7847e4cd55c. Thanks again!

**Pull request closed without merging**